### PR TITLE
First batch of NLU signals

### PR DESCRIPTION
**Pull request by bfilar**

### CODE REVIEW

File: signals/content/nlu_bec_high_conf.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: "NLU Intent: Business Email Compromise"
```
Comment (jkamdjou) [inline signals/content/nlu_bec_high_conf.yml:1]: do u want the confidence in the name like the others? missing on a few of em

File: signals/content/nlu_bec_high_conf.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+type: "query"
+source: |
+  any(ml.nlu_classifier(coalesce(body.html.display_text, body.plain.raw)).intents, .name=="bec" and .confidence == "high")
+
```
Comment (jkamdjou) [inline signals/content/nlu_bec_high_conf.yml:5]: extra new line here and in some others

**Review by jkamdjou — COMMENTED** [review]
l f g

File: signals/content/nlu_bec_high_conf.yml (hunk 1 above)
Comment (bfilar) [reply signals/content/nlu_bec_high_conf.yml:1]: yeah good call.

**Comment by bfilar** [comment]
Ready for review

**Comment by morriscode** [comment]
These look good to me. The only thing I question is the use of Tags. None of the other signals have any tags on them, I'm not sure if those actually do anything, or if they're helpful in your model, or even if they work on signals, I assumed they would as it's probably the same schema, but I don't know.

**Comment by bfilar** [comment]
that's a fair point. i should probably add them

**Comment by bfilar** [comment]
Oh I see what you're saying. Let's get them in as they could be useful indicators when observed with other signals

**Comment by bfilar** [comment]
Based on internal testing from @rw-access this should be good to merge. Someone review & give a 👍

**Comment by rw-access** [comment]
FWIW this PR wasn't blocked before testing and isn't now, so don't let me hold you up!

**Review by morriscode — APPROVED** [review]
:shipit:

**Pull request merged**